src/utils/util.ts

```diff
@@ -0,0 +1,8 @@
+export function shuffleArray(arr: string[]) {
+  const array = [...arr];
+  for (let i = array.length - 1; i > 0; i--) {
+    const newIndex = Math.floor(Math.random() * array.length);
+    [array[i], array[newIndex]] = [array[newIndex], array[i]];
```

Comment:
It's a knit-pick, but `i` and `newIndex` look like an inconsistent naming. Either name `i` > `index`, or `newIndex` > `newI`